debug/terrain-debug.html

```diff
@@ -177,11 +178,87 @@
             ]
         }
     });
+
+    var highlightLayer = {
```

Comment:
Added terrain-debug demo with a new custom layer for manual testing and debugging.